src/platform/console/unix_console.h

```diff
@@ -36,16 +42,18 @@ class UnixConsole final : public Console
     };
     void write_console(const char* buffer, int bytes) override{};
     void signal_console() override{};
-    bool is_window_size_changed() override;
-    bool is_interactive() override;
-    WindowGeometry get_window_geometry() override;
+
+    static void setup_environment();
 
 private:
     void setup_console() override;
     void restore_console() override;
 
     bool interactive{false};
     struct termios saved_terminal;
+    ssh_channel channel;
```

Comment:
Clang complains that channel is unused:

> In file included from ../src/platform/console/unix_console.cpp:18:
../src/platform/console/unix_console.h:54:17: error: private field 'channel' is not used [-Werror,-Wunused-private-field]
    ssh_channel channel;